assets/js/modules/analytics/datastore/profiles.test.js

```diff
@@ -155,6 +155,7 @@ describe( 'modules/analytics profiles', () => {
 	describe( 'selectors', () => {
 		describe( 'getProfiles', () => {
 			it( 'uses a resolver to make a network request', async () => {
+				registry.dispatch( STORE_NAME ).setSettings( {} );
```

Comment:
Why did we need to add this? Is it likely something we'll need in all tests? It just seems a bit out-of-place and there's no comment explaining it.

Reply:
Yeah, I think there's probably an enhancement we could add with a `beforeEach`. Basically, it messes up expectations on the fetch mock without it because settings are resolved from a fetch request if they're not set. I'm open to ideas as to how we should best solve this 🙂